### ISSUE TITLE
MEmodel update

### ISSUE THREAD

**Issue by zq130320339** [caoyunkang/CDO#11]
Hi, have you tested the effect of MEmdel doing gradient update vs. no update, I tested that the difference is not that much

**Comment by caoyunkang** [comment]
Hi, I have not tried this. However, it may be reasonable to update it in the first several epochs, so that the ME model is adapted to the target category, mitigating the domain gap. Just be careful that too much training on the ME model may lead to overfitting. Best.

**Comment by zq130320339** [comment]
Thanks for the reply, I tried adding some supervised data training, but the test was not very good, some images in the test set that were similar to the training performed poorly.

**Comment by caoyunkang** [comment]
Can you describe your problem more clearly? Try to use different backbones and the performance may vary.
